show/main.py

```diff
@@ -1865,6 +1865,53 @@ def table(table_name, verbose):
 
     run_command(cmd, display_cmd=verbose)
 
+#
+# 'drops' group ###
+#
+
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def drops():
+    """Show drop counter related information"""
+    pass
+
+# 'config' subcommand ("show drops config")
+@drops.command()
+@click.option('-g', '--group', required=False)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def config(group, verbose):
```

Comment:
Suggest to rename command using full word, "configuration", as the Click library will automatically handle tab-completion and abbreviation.